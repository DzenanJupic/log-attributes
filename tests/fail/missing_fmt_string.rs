#[log_attributes::log(info)]
fn missing_fmt_string() {}

fn main() {}
