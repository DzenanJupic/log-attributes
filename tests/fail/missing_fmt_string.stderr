error: No format string supplied
       help: log requires a fmt string that describes the log output
       example: #[log(info, "{fn} returned {return}")]
 --> tests/fail/missing_fmt_string.rs:1:1
  |
1 | #[log_attributes::log(info)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `log_attributes::log` (in Nightly builds, run with -Z macro-backtrace for more info)
